e2e/playwright/test-utils.ts

```diff
@@ -522,23 +522,23 @@ export async function getUtils(page: Page, test_?: typeof test) {
     createNewFile: async (name: string) => {
       return test?.step(`Create a file named ${name}`, async () => {
         await page.getByTestId('create-file-button').click()
-        await page.getByTestId('tree-input-field').fill(name)
+        await page.getByTestId('file-rename-field').fill(name)
         await page.keyboard.press('Enter')
       })
     },
 
     createNewFolder: async (name: string) => {
       return test?.step(`Create a folder named ${name}`, async () => {
         await page.getByTestId('create-folder-button').click()
-        await page.getByTestId('tree-input-field').fill(name)
+        await page.getByTestId('file-rename-field').fill(name)
         await page.keyboard.press('Enter')
       })
     },
 
     cloneFile: async (name: string) => {
       return test?.step(`Cloning file '${name}'`, async () => {
         await page
-          .locator('[data-testid="file-pane-scroll-container"] button')
+          .locator('[data-testid="file-pane-scroll-container"] [role=treeitem]')
```

Comment:
The DOM element is not a button anymore, I can use `role=treeitem` instead. This is a reasonable change across the helper functions.